packages/beachball/src/__e2e__/publishE2E.test.ts

```diff
@@ -62,6 +62,7 @@ describe('publish command (e2e)', () => {
       bumpDeps: true,
       push: true,
       registry: registry.getUrl(),
+      autoTag: true,
```

Comment:
Per an offline discussion: I think we should have this be called "gitTag" and by default it should be `undefined` which would mean that it would automatically be using "tag". In the future, we'll fill this API out with a function `(pkg, version) => string` that will generate customizable git tags.

Reply:
renamed to `gitTags`  with cli aliases `--git-tags` and `--no-git-tags`